docs/hosting/installation/server-setups/docker-compose.md

```diff
@@ -4,24 +4,18 @@ If you have already installed Docker and Docker-Compose, then you can start with
 
 ### 1. Install Docker
 
-This can vary depending on the Linux distribution used. The below example is for Ubuntu:
+This can vary depending on the Linux distribution used. You can find detailed instructions [here in the Docker documentation](https://docs.docker.com/engine/install/). The below example is for Ubuntu:
```

Comment:
```suggestion
This can vary depending on the Linux distribution used. You can find detailed instructions in the [Docker documentation](https://docs.docker.com/engine/install/){:target=_blank .external-link}. The following example is for Ubuntu:
```